src/application/window.rs

```
@@ -714,13 +714,14 @@ impl Window {
     self.maximized.load(Ordering::Acquire)
   }
 
-  pub fn drag_window(&self) {
+  pub fn drag_window(&self) -> Result<(), ExternalError> {
     if let Err(e) = self
```

should we propagate this error instead?

---

why ? `ExternalError` indicates its either a OS error, or just not a supported method and the error here is `SendError`.

---

yeah, and also i think the SendError should never happen right

---

SendError occurs only when the receiver is disconnected first. This depends on how gtk handles its threads and tasks.